test/foo.test.js

```diff
@@ -1,13 +1,12 @@
 /*global describe, it*/
-'use strict';
 
-var expect = require('expect.js');
+const expect = require('expect.js');
 
-var Foo = require('./../src/js/foo');
+const foo = require('./../src/js/foo');
```

Comment:
It's just a component blueprint, but we should be thinking that if it's a module, for it to act like a constructor. Feel free to improve the blueprint to show that behaviour